modules/ROOT/pages/visualizer-app-network.adoc

```diff
@@ -59,6 +59,8 @@ To avoid long numbers, Anypoint Visualizer uses unit prefixes to simplify your v
 |===
 https://en.wikipedia.org/wiki/Metric_prefix[Here is a quick review of metric prefixes], but note that, in Anypoint Visualizer, *u* represents micro, rather than *μ*.
 
+Metrics are also available for connections, or _edges_, in your network. When you select average response time or average throughput, metrics also appear on the related connection. For more information on which metrics are available, see xref:connection-metrics-compatibility.adoc[Supported Connection-Level Metrics].
```

Comment:
I would change "metrics also appear on the related connection" to "metrics also appear on applicable connections." Also, what are the best practices for linking to other pages in the docs? I think we can remove the link here and just keep it in the following section.

Reply:
```suggestion
Metrics are also available for connections, or _edges_, in your network. When you select average response time or average throughput, metrics also appear on applicable connections. 
```